CHANGELOG.md

```diff
@@ -5,6 +5,7 @@
 - On Windows, fix bug causing mouse capture to not be released.
 - On Windows, fix fullscreen not preserving minimized/maximized state.
 - On Android, unimplemented events are marked as unhandled on the native event loop.
+- On Web, added `DeviceEvent::MouseMotion` to support use with Pointer Lock API.
```

Comment:
I'm not really sure if it makes sense to mention "Pointer Lock API" here. This may give the wrong impression that Winit supports it, but it is yet to be implemented...

Reply:
Ok, I removed the reference to pointer lock.